qmcpy/integrand/um_bridge_wrapper.py

```diff
@@ -0,0 +1,51 @@
+from ._integrand import Integrand
+from ..util import ParameterError
+from numpy import *
+
+class UMBridgeWrapper(Integrand):
+    """
+    UM-Bridge Model Wrapper
+    References:
+
+        [1] UM-Bridge documentation. https://um-bridge-benchmarks.readthedocs.io/en/docs/index.html
+    """
+
+    def __init__(self, true_measure, model, config, parallel=False):
+        """
+        See https://um-bridge-benchmarks.readthedocs.io/en/docs/umbridge/clients.html
+        
+        Args:
+            true_measure (TrueMeasure): a TrueMeasure instance. 
+            model (umbridge.HTTPModel): a UM-Bridge model 
+            config (dict): config keyword argument to umbridge.HTTPModel(url,name).__call__
+            parallel (int): If parallel is False, 0, or 1: function evaluation is done in serial fashion.
+                Otherwise, parallel specifies the number of processes used by 
+                multiprocessing.Pool or multiprocessing.pool.ThreadPool.
+                Passing parallel=True sets processes = os.cpu_count().
+        """
+        import umbridge
+        self.parameters = []
+        self.true_measure = true_measure
+        self.sampler = self.true_measure 
+        self.model = model
```

Comment:
You can check for `model.supports_evaluate()` to be sure the model allows direct evaluation (each "feature" of an umbridge model is technically optional. That makes a lot more sense for e.g. Hessian action, but technically there could be models not supporting evaluation, and it would be cleaner to catch that here)

Reply:
Makes sense. I've updated so an error is raised `if not model.supports_evalute()`.